nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/AbstractElasticsearchProcessor.java

```diff
@@ -28,14 +29,20 @@
 
 import java.util.Collection;
 import java.util.HashSet;
-import java.util.Map;
 import java.util.Set;
 
 /**
  * A base class for all Elasticsearch processors
  */
 public abstract class AbstractElasticsearchProcessor extends AbstractProcessor {
 
+    protected static final Validator NON_EMPTY_EL_VALIDATOR = (subject, value, context) -> {
```

Comment:
Yes will do. The Elasticsearch 5 bundle has this too, should I move them as part of a new Jira, or allow this PR to touch that bundle for this purpose?